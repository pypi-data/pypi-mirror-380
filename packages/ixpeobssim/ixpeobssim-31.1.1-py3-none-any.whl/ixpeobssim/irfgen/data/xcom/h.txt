#Photon    Coherent Incoher. Photoel.
#Energy    Scatter. Scatter. Absorb.
#
1.000E-03 3.468E-01 5.033E-02 6.817E+00
1.028E-03 3.439E-01 5.277E-02 6.215E+00
1.056E-03 3.410E-01 5.526E-02 5.681E+00
1.086E-03 3.380E-01 5.797E-02 5.173E+00
1.116E-03 3.350E-01 6.073E-02 4.722E+00
1.147E-03 3.320E-01 6.363E-02 4.308E+00
1.178E-03 3.290E-01 6.656E-02 3.940E+00
1.211E-03 3.259E-01 6.973E-02 3.592E+00
1.245E-03 3.226E-01 7.303E-02 3.274E+00
1.279E-03 3.194E-01 7.637E-02 2.991E+00
1.315E-03 3.159E-01 7.994E-02 2.725E+00
1.351E-03 3.125E-01 8.353E-02 2.489E+00
1.389E-03 3.088E-01 8.735E-02 2.268E+00
1.427E-03 3.051E-01 9.119E-02 2.072E+00
1.467E-03 3.011E-01 9.524E-02 1.888E+00
1.507E-03 2.971E-01 9.929E-02 1.725E+00
1.549E-03 2.929E-01 1.035E-01 1.572E+00
1.592E-03 2.885E-01 1.079E-01 1.434E+00
1.636E-03 2.841E-01 1.123E-01 1.308E+00
1.682E-03 2.794E-01 1.170E-01 1.191E+00
1.728E-03 2.747E-01 1.216E-01 1.088E+00
1.776E-03 2.698E-01 1.263E-01 9.917E-01
1.825E-03 2.648E-01 1.312E-01 9.046E-01
1.876E-03 2.597E-01 1.362E-01 8.242E-01
1.928E-03 2.545E-01 1.412E-01 7.514E-01
1.981E-03 2.493E-01 1.463E-01 6.856E-01
2.036E-03 2.439E-01 1.514E-01 6.249E-01
2.093E-03 2.385E-01 1.567E-01 5.691E-01
2.151E-03 2.330E-01 1.621E-01 5.188E-01
2.211E-03 2.274E-01 1.675E-01 4.726E-01
2.272E-03 2.219E-01 1.728E-01 4.310E-01
2.335E-03 2.163E-01 1.783E-01 3.928E-01
2.400E-03 2.107E-01 1.838E-01 3.578E-01
2.466E-03 2.051E-01 1.892E-01 3.264E-01
2.535E-03 1.994E-01 1.948E-01 2.972E-01
2.605E-03 1.938E-01 2.003E-01 2.709E-01
2.677E-03 1.882E-01 2.058E-01 2.469E-01
2.751E-03 1.826E-01 2.112E-01 2.251E-01
2.828E-03 1.770E-01 2.167E-01 2.049E-01
2.906E-03 1.715E-01 2.221E-01 1.868E-01
2.987E-03 1.660E-01 2.275E-01 1.701E-01
3.070E-03 1.606E-01 2.328E-01 1.550E-01
3.155E-03 1.552E-01 2.380E-01 1.412E-01
3.242E-03 1.499E-01 2.431E-01 1.288E-01
3.332E-03 1.447E-01 2.482E-01 1.173E-01
3.424E-03 1.396E-01 2.531E-01 1.069E-01
3.519E-03 1.345E-01 2.580E-01 9.739E-02
3.617E-03 1.296E-01 2.628E-01 8.869E-02
3.717E-03 1.248E-01 2.674E-01 8.082E-02
3.820E-03 1.201E-01 2.720E-01 7.362E-02
3.926E-03 1.155E-01 2.764E-01 6.706E-02
4.035E-03 1.110E-01 2.807E-01 6.107E-02
4.147E-03 1.066E-01 2.848E-01 5.562E-02
4.262E-03 1.024E-01 2.889E-01 5.065E-02
4.380E-03 9.827E-02 2.928E-01 4.613E-02
4.502E-03 9.427E-02 2.966E-01 4.199E-02
4.627E-03 9.040E-02 3.002E-01 3.824E-02
4.755E-03 8.667E-02 3.038E-01 3.482E-02
4.887E-03 8.305E-02 3.072E-01 3.171E-02
5.022E-03 7.957E-02 3.104E-01 2.888E-02
5.162E-03 7.617E-02 3.136E-01 2.628E-02
5.305E-03 7.292E-02 3.166E-01 2.393E-02
5.452E-03 6.978E-02 3.195E-01 2.179E-02
5.603E-03 6.676E-02 3.223E-01 1.984E-02
5.758E-03 6.385E-02 3.250E-01 1.807E-02
5.918E-03 6.105E-02 3.275E-01 1.645E-02
6.082E-03 5.835E-02 3.300E-01 1.498E-02
6.251E-03 5.575E-02 3.323E-01 1.364E-02
6.424E-03 5.327E-02 3.346E-01 1.243E-02
6.602E-03 5.088E-02 3.367E-01 1.132E-02
6.785E-03 4.858E-02 3.387E-01 1.031E-02
6.974E-03 4.637E-02 3.407E-01 9.382E-03
7.167E-03 4.426E-02 3.425E-01 8.546E-03
7.366E-03 4.223E-02 3.442E-01 7.782E-03
7.570E-03 4.029E-02 3.459E-01 7.087E-03
7.780E-03 3.842E-02 3.474E-01 6.453E-03
7.996E-03 3.663E-02 3.489E-01 5.875E-03
8.217E-03 3.493E-02 3.503E-01 5.350E-03
8.445E-03 3.329E-02 3.516E-01 4.870E-03
8.680E-03 3.171E-02 3.528E-01 4.432E-03
8.920E-03 3.022E-02 3.540E-01 4.035E-03
9.168E-03 2.878E-02 3.551E-01 3.672E-03
9.422E-03 2.741E-02 3.561E-01 3.343E-03
9.683E-03 2.609E-02 3.570E-01 3.043E-03
9.952E-03 2.484E-02 3.579E-01 2.769E-03
1.023E-02 2.364E-02 3.587E-01 2.520E-03
1.051E-02 2.249E-02 3.595E-01 2.294E-03
1.080E-02 2.140E-02 3.602E-01 2.087E-03
1.110E-02 2.035E-02 3.608E-01 1.900E-03
1.141E-02 1.936E-02 3.614E-01 1.729E-03
1.173E-02 1.840E-02 3.619E-01 1.574E-03
1.205E-02 1.750E-02 3.623E-01 1.432E-03
1.239E-02 1.663E-02 3.627E-01 1.304E-03
1.273E-02 1.580E-02 3.631E-01 1.186E-03
1.308E-02 1.502E-02 3.634E-01 1.080E-03
1.344E-02 1.427E-02 3.636E-01 9.827E-04
1.382E-02 1.355E-02 3.638E-01 8.943E-04
1.420E-02 1.287E-02 3.640E-01 8.139E-04
1.459E-02 1.223E-02 3.641E-01 7.406E-04
1.500E-02 1.161E-02 3.642E-01 6.739E-04
