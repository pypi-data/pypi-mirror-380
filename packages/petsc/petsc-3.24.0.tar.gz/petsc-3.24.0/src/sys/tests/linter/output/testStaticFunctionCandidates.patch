@@ -4,5 +4,5 @@
 #include "testheader.h"
 
-void BareFunctionShouldGetStatic(void) { }
+static void BareFunctionShouldGetStatic(void) { }
 
 extern void ExternFunctionShouldNotGetStatic(void) { }
@@ -14,5 +14,5 @@
 
 // this should get static!
-void StaticFunctionPreDeclShouldNotGetStatic(void) { }
+static void StaticFunctionPreDeclShouldNotGetStatic(void) { }
 
 extern void ExternFunctionPreDeclShouldNotGetStatic(void);
@@ -20,5 +20,5 @@
 void ExternFunctionPreDeclShouldNotGetStatic(void) { }
 
-void BareFunctionPreDeclShouldGetStatic(void);
+static void BareFunctionPreDeclShouldGetStatic(void);
 
 void BareFunctionPreDeclShouldGetStatic(void) { }
@@ -22,5 +22,5 @@
 void BareFunctionPreDeclShouldGetStatic(void);
 
-void BareFunctionPreDeclShouldGetStatic(void) { }
+static void BareFunctionPreDeclShouldGetStatic(void) { }
 
 // declaration in testheader has "extern"
@@ -46,5 +46,5 @@
 }
 
-char *BarePointerShouldGetStatic()
+static char *BarePointerShouldGetStatic()
 {
   return nullptr;
@@ -99,5 +99,5 @@
 
 // ironically enough, this will get static
-void silence_warnings(void)
+static void silence_warnings(void)
 {
   (void)StaticFunctionShouldNotGetStatic;
