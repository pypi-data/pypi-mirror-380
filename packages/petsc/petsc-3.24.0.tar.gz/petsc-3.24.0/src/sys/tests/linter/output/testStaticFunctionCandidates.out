<--- FIXED --->
[31m[1m./src/sys/tests/linter/testStaticFunctionCandidates.cxx:3:1: error:[0m Function 'BareFunctionShouldGetStatic()' does not appear to be used anywhere outside of testStaticFunctionCandidates.cxx, and can be made static [-fstatic-function-candidate]:
  1: #include "testheader.h"
  2: 
> 3: [33m[1mvoid BareFunctionShouldGetStatic(void) { }[0m
     [33m[1m^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
  4: 
  5: extern void ExternFunctionShouldNotGetStatic(void) { }
[31m[1m./src/sys/tests/linter/testStaticFunctionCandidates.cxx:13:1: error:[0m Function 'StaticFunctionPreDeclShouldNotGetStatic()' does not appear to be used anywhere outside of testStaticFunctionCandidates.cxx, and can be made static [-fstatic-function-candidate]:
  11: 
  12: // this should get static!
> 13: [33m[1mvoid StaticFunctionPreDeclShouldNotGetStatic(void) { }[0m
      [33m[1m^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
  14: 
  15: extern void ExternFunctionPreDeclShouldNotGetStatic(void);
[31m[1m./src/sys/tests/linter/testStaticFunctionCandidates.cxx:19:1: error:[0m Function 'BareFunctionPreDeclShouldGetStatic()' does not appear to be used anywhere outside of testStaticFunctionCandidates.cxx, and can be made static [-fstatic-function-candidate]:
  17: void ExternFunctionPreDeclShouldNotGetStatic(void) { }
  18: 
> 19: [33m[1mvoid BareFunctionPreDeclShouldGetStatic(void);[0m
      [33m[1m^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
  20: 
  21: void BareFunctionPreDeclShouldGetStatic(void) { }
[31m[1m./src/sys/tests/linter/testStaticFunctionCandidates.cxx:21:1: error:[0m Function 'BareFunctionPreDeclShouldGetStatic()' does not appear to be used anywhere outside of testStaticFunctionCandidates.cxx, and can be made static [-fstatic-function-candidate]:
  19: void BareFunctionPreDeclShouldGetStatic(void);
  20: 
> 21: [33m[1mvoid BareFunctionPreDeclShouldGetStatic(void) { }[0m
      [33m[1m^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
  22: 
  23: // declaration in testheader has "extern"
[31m[1m./src/sys/tests/linter/testStaticFunctionCandidates.cxx:45:1: error:[0m Function 'BarePointerShouldGetStatic()' does not appear to be used anywhere outside of testStaticFunctionCandidates.cxx, and can be made static [-fstatic-function-candidate]:
  43: }
  44: 
> 45: [33m[1mchar *BarePointerShouldGetStatic()[0m
      [33m[1m^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
  46: {
  47:   return nullptr;
[31m[1m./src/sys/tests/linter/testStaticFunctionCandidates.cxx:98:1: error:[0m Function 'silence_warnings()' does not appear to be used anywhere outside of testStaticFunctionCandidates.cxx, and can be made static [-fstatic-function-candidate]:
  96 : 
  97 : // ironically enough, this will get static
> 98 : [33m[1mvoid silence_warnings(void)[0m
       [33m[1m^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
  99 : {
  100:   (void)StaticFunctionShouldNotGetStatic;
<--- LEFT --->
